chia/pools/pool_wallet.py

```diff
@@ -572,14 +566,14 @@ async def generate_travel_transactions(self, fee: uint64) -> Tuple[TransactionRe
         else:
             raise RuntimeError("Invalid state")
 
-        fee_tx = None
-        if fee > 0:
-            fee_tx = await self.generate_fee_transaction(fee)
-
         signed_spend_bundle = await self.sign(outgoing_coin_spend)
         assert signed_spend_bundle.removals()[0].puzzle_hash == singleton.puzzle_hash
         assert signed_spend_bundle.removals()[0].name() == singleton.name()
         assert signed_spend_bundle is not None
+        fee_tx = None
+        if fee > 0:
+            fee_tx = await self.generate_fee_transaction(fee)
+            signed_spend_bundle = SpendBundle.aggregate([signed_spend_bundle, fee_tx.spend_bundle])
```

Comment:
Thank you. I had this aggregate in an earlier iteration, and I missed it while making other changes.